website/content/docs/other-specifications/volume/index.mdx

```diff
@@ -64,255 +62,5 @@ parameters {
-
-Nomad will reconcile the requested capacity by issuing expand volume requests
-to the controller plugin, and if required by the controller, also to the
-node plugins for each allocation that has a claim on the volume.
-
-## Examples
-
-### Volume registration
-
-This is an example file used for the [`volume register`] command.
-
-```hcl
-id              = "ebs_prod_db1"
-name            = "database"
-type            = "csi"
-external_id     = "vol-23452345"
-plugin_id       = "ebs-prod"
-
-capability {
-  access_mode     = "single-node-reader-only"
-  attachment_mode = "file-system"
-}
-
-capability {
-  access_mode     = "single-node-writer"
-  attachment_mode = "file-system"
-}
-
-mount_options {
-  fs_type     = "ext4"
-  mount_flags = ["noatime"]
-}
-
-topology_request {
-  required {
-    topology { segments { "rack" = "R2" } }
-    topology { segments { "rack" = "R1", "zone" = "us-east-1a"} }
-  }
-}
-
-secrets {
-  example_secret = "xyzzy"
-}
-
-parameters {
-  skuname = "Premium_LRS"
-}
-
-context {
-  endpoint = "http://192.168.1.101:9425"
-}
-```
-
-
-### Example Volume Expansion
-
-Either [`volume create`][] or [`volume register`][] commands can trigger
-an expand to occur, after the volume has already been created or registered.
-
-Example using `volume create`, with a `volume.hcl` file:
-
-```hcl
-id        = "ebs_prod_db1"
-name      = "database"
-type      = "csi"
-plugin_id = "ebs-prod"
-
-capacity_min = "50gib"
-capacity_max = "50gib"
-
-# ... etc ...
-```
-
-Create the volume:
-
-```shell-session
-$ nomad volume create volume.hcl
-Created external volume vol-asdf1234 with ID ebs_prod_db1
-```
-
-See its current capacity with [`volume status`][]:
-
-```shell-session
-$ nomad volume status ebs_prod_db1 | grep Capacity
-Capacity             = 50 GiB
-```
-
-Update `volume.hcl`:
-
-```hcl
-id        = "ebs_prod_db1"
-name      = "database"
-type      = "csi"
-plugin_id = "ebs-prod"
-# all of the above must remain the same
-
-capacity_min = "100gib" # double
-capacity_max = "100gib" # increased to match
-
-# ... etc ...
-```
-
-Run `volume create` again:
-
-```shell-session
-$ nomad volume create volume.hcl
-Created external volume vol-asdf1234 with ID ebs_prod_db1
-```
-
-Check the volume capacity again:
-
-```shell-session
-$ nomad volume status ebs_prod_db1 | grep Capacity
-Capacity             = 100 GiB
-```
-
-If you encounter errors that are not clear from the command output,
-the CSI plugin allocation logs and/or Nomad leader server logs may be
-helpful.
-
-[api_volume_create]: /nomad/api-docs/volumes#create-csi-volume
-[api_volume_register]: /nomad/api-docs/volumes#register-csi-volume
-[capability]: /nomad/docs/other-specifications/volume/capability
-[csi_plugin]: /nomad/docs/job-specification/csi_plugin
-[csi_volume_source]: /nomad/docs/job-specification/volume#source
-[mount_options]: /nomad/docs/other-specifications/volume/mount_options
-[topology_request]: /nomad/docs/other-specifications/volume/topology_request
```

Comment:
since we're not supporting `mount_options` nor `topology_request` for DHVs, I think the top descriptions in those sub-pages should say that they are for CSI only, so DHV readers may know quickly that those pages are not relevant for them.  Their being simply excluded from the new DHV spec page is fine, but we should be explicit about it on the sub-pages, since folks may get there from our side nav, or search engines.